pkg/sidecar-controller/groupsnapshot_helper.go

```diff
@@ -802,7 +802,18 @@ func (ctrl *csiSnapshotSideCarController) checkandUpdateGroupSnapshotContentStat
 			}
 		}
 
-		readyToUse, creationTime, err = ctrl.handler.GetGroupSnapshotStatus(groupSnapshotContent, snapshotterListCredentials)
+		var snapshotIDs []string
+		if groupSnapshotContent.Status != nil && len(groupSnapshotContent.Status.VolumeSnapshotContentRefList) != 0 {
```

Comment:
Is there a possibility that this piece of code is hit before `groupSnapshotContent.Status` is populated, and therefore the `GetGroupSnapshotStatus` has an empty list of snapshotIDs and will fail. What happens then?

Reply:
Maybe, not sure. But I'll use `groupSnapshotContent.Spec.Source.GroupSnapshotHandles.VolumeSnapshotHandles` instead, that was added with the recent API change.